.github/workflows/deps.yml

```diff
@@ -0,0 +1,55 @@
+name: Check Spack dependencies for updates
+
+on:
+  # for manual jobs
+  workflow_dispatch:
+  # triggers at around 15:00 UTC (with some delay)
+  schedule:
+   - cron: '0 15 * * *'
+
+jobs:
+    deps:
+      name:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout this repo
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+        - name: create build
+          run: mkdir build
+        - name: Checkout ports-of-call
+          uses: actions/checkout@v3
+          with:
+            repository: lanl/ports-of-call
+            path: build/ports-of-call
+            fetch-depth: 0
+        - name: Checkout spiner
+          uses: actions/checkout@v3
+          with:
+            repository: lanl/spiner
+            path: build/spiner
+            fetch-depth: 0
+        - name: copy over spack-repo/packages
+          run: |
+            # assume ports-of-call may be newer than spiner
+            cp -R build/spiner/spack-repo/packages/* spack-repo/packages/
+            cp -R build/ports-of-call/spack-repo/packages/* spack-repo/packages/
+        - name: is there a difference?
+          run: git diff --exit-code --compact-summary
+          id: no_change
+        - name: create branch
+          if: ${{ failure() && steps.no_change.conclusion == 'failure' }}
```

Comment:
Forgive my ignorance, but what is the purpose of `failure()` here?

Reply:
just a copy&paste thing. It shouldn't be necessary, so I removed it.